web/migrations/versions/add_user_id_to_debugger_func_args_.py

```
@@ -0,0 +1,123 @@
+        'CREATE INDEX IF NOT EXISTS ix_sharedserver_user_id '
+        'ON sharedserver (user_id)',
+        'CREATE INDEX IF NOT EXISTS ix_sharedserver_osid '
+        'ON sharedserver (osid)',
+        'CREATE INDEX IF NOT EXISTS ix_servergroup_user_id '
+        'ON servergroup (user_id)',
+    ]
+    for stmt in index_stmts:
+        try:
+            op.execute(stmt)
+        except Exception:
+            pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)